axelrod/strategies/titfortat.py

```diff
@@ -300,7 +300,7 @@ def strategy(self, opponent: Player) -> Action:
             return C
         # TFT on round 2
         if len(self.history) == 1:
-            return D if opponent.history[-1:] == [D] else C
```

Comment:
I think we use this because that's how TFT is actually implemented in the library, where we don't know that the history length is greater than zero. I don't have a preference here and I think your line might be more readable, but we might prefer consistency. So I'm approving and I'll let @drvinceknight and @meatballs weigh in if they have a preference.